### PR TITLE
ON-15428: README.md for deployment with sample overlays

### PR DESCRIPTION
**Pull request by pcolledg-amd**
Deployment-focused README.md and sample overlays. I was not intending to add overlays so soon but it does make instructions super simple in comparison to what we had before.

I will shortly be integrating:
- [x] Link to forthcoming `config/samples/sfptpd/README.md`
- [ ] Updates from Day 0 addendum changes
- [x] Caveats (brief initial list only)
- [ ] Overlays suitable for our dev (optional, but probably easy)

The rest can come in subsequent commits. I'll harmonise with Release Notes, glaring differences only.

# Please ignore BUILD.md for now
It is untouched, I'll rebase it before merging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,247 +2,148 @@
...
 
-6. Export the following Onload information for tagging the Onload kernel image.
-```text
-export ONLOAD_VERSION=<onload-tag>
-export ONLOAD_KERNEL_VERSION=<kernel version to be built>
-```
-n.b. `onload-tag` can be of the form of a release e.g. `v8.1.0` or a git commit
-hash, and `onload_kernel_version` is expected to be of the form
-`4.18.0-372.49.1.el8_6.x86_64`.
+The cluster also requires access to the following node-specific kernel module container image(s) which may be provided externally or internally. If using in-cluster builds, push access to an internal registry will be required, otherwise only pull access is required if these images are pre-built. Pre-built images are planned in the next release for EnterpriseOnload customers.
```
Comment (abower-amd) [inline README.md:46]: Last sentence is out of scope for the README - needs removing.

**Review by abower-amd — CHANGES_REQUESTED** [review]
Some required changes identified but I like this.

**Review by jjay-xilinx — COMMENTED** [review]
beside the nitpick of headings, readme looks fine

**Review by abower-amd — APPROVED** [review]
Great thanks, gives good shape.

**Pull request merged**